m4opt/synphot/background/_cerenkov/_refraction_index.py

```diff
@@ -0,0 +1,280 @@
+"""
+Python conversion of the MATLAB script `refraction_index.m` by Eran Ofek,
+available at:
+https://github.com/EranOfek/AstroPack
+
+This module computes the refractive index and transmission for optical
+materials as a function of wavelength.
+"""
+
+import astropy.units as u
+import numpy as np
+
+
+def get_refraction_index(material="si02_suprasil_2a", temperature=20):
+    """
+    Compute refractive index and transmission for a given material.
+    See module docstring for context and references.
+
+    Parameters
+    ----------
+    material : str, optional
+        Supported options:
+        - 'sio2', 'silica' : fused silica (Sellmeier equation)
+        - 'si02_suprasil_2a' : Suprasil 2A (tabulated data)
+        Default: 'si02_suprasil_2a'.
+
+    temperature : float, optional
+        Temperature in Celsius. Only 20°C supported.
+
+    Returns
+    -------
+    L : np.ndarray
+        Wavelengths [Å].
+    n : np.ndarray
+        Refractive index.
+    t : np.ndarray or None
+        Transmission per 1 cm (only for Suprasil 2A), or None.
+
+    Raises
+    ------
+    ValueError
+        For unsupported material or temperature.
+    """
+
+    material = material.lower()
+    if material in ["sio2", "silica"]:
```

Comment:
- SiO2 is the chemical formula for silica. Do you really need to support both strings, `sio2` and `silica`?
- This function has very little in common across the three `if`/`elif` paths. Please refactor into three separate functions. If you want to provide a high-level function that provides a string-based lookup, that's fine, but please do that using a dictionary lookup.